docs/docs/guides/labs/components/building-pipelines-with-components/adding-attributes-to-assets.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Adding attributes to assets in a subdirectory'
+sidebar_position: 550
+---
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+Attaching and modifying meta information on definitions such as assets is important in Dagster projects. You want to have groups, teams, owners, kinds, tags, and metadata set correctly to organize definitions and ensure that other tools and processes that rely on them correctly function.
+
+Within a dg-driven `defs` project layout, you can apply metadata transformations at any point in the folder structure. This can span uses cases from ensuring that all definitions in a particular folder have a tag associating them with the same team, to more complex workflows of applying a group conditionally based on other properties of the definition.
```

Comment:
"to more complex use cases."

The "of applying a group conditionally based on other properties of the definition." makes it sounds like it is the only advanced use case we support. 

We can add examples lower in the document of more complex use cases. We should have copy-pasteable stuff.